{
  "ID": "pyup.io-40001",
  "Advisory": "Tensorflow 2.1.3 fixes a security vulnerability caused by accessing heap data outside of bounds when loading a specially crafted 'SavedModel' (CVE-2020-26271).",
  "Cve": "CVE-2020-26271",
  "Specs": [
    ">=2.1.0,<2.1.3"
  ],
  "v": ">=2.1.0,<2.1.3"
}
